### ISSUE TITLE
Documentation on how to change website favicon

### ISSUE THREAD

**Issue by StrangeMcCarthy** [varabyte/kobweb#489]
*ADDITIONAL DETAILS HERE*

---

(Optional) The relevant location:

- [ ] README
- [ ] Code comments
- [ ] Website docs (https://kobweb.varabyte.com)

(Mark any with an `x`)

**Comment by bitspittle** [comment]
Sure, I'll think about the best place for it.

If you need an answer ASAP, go into the site created by `kobweb create app` (or any of the other ones) and find a favicon icon file in the `jsMain/resources/public` folder. Then change it to whatever you'd like, and it should work as long as it's a valid icon file.

**Comment by StrangeMcCarthy** [comment]
I did that already but it's still not working for me it still displays the
kobweb icon

**Comment by bitspittle** [comment]
Did you hit shift+refresh?

**Comment by StrangeMcCarthy** [comment]
Yes, I did.

**Comment by StrangeMcCarthy** [comment]
But I figured it out already thanks 🙏

**Comment by bitspittle** [comment]
So you're fixed? It sounds like you figured out what to do but just had caching issues with the browser?

**Comment by StrangeMcCarthy** [comment]
Yeah that's itSent from my Huawei Mobile-------- Original Message --------Subject: Re: [varabyte/kobweb] Documentation on how to change website favicon (Issue #489)From: David Herman To: varabyte/kobweb CC: StrangeMcCarthy ,Author 
So you're fixed? It sounds like you figured out what to do but just had caching issues with the browser?

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>
[
{
***@***.***": "http://schema.org",
***@***.***": "EmailMessage",
"potentialAction": {
***@***.***": "ViewAction",
"target": "https://github.com/varabyte/kobweb/issues/489#issuecomment-2028937433",
"url": "https://github.com/varabyte/kobweb/issues/489#issuecomment-2028937433",
"name": "View Issue"
},
"description": "View this Issue on GitHub",
"publisher": {
***@***.***": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]

**Comment by StrangeMcCarthy** [comment]
Thanks alot for adding it into the repository
